specification/common-types/resource-management/v2/clouderror.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Why we need this file added here? We already have `ErrorResponse` and `ErrorDetails` here which are exactly the same as the things in this file: https://github.com/Azure/azure-rest-api-specs/blob/1b3b9c1dd4d2c875997ea0b392dc71418fb1f28d/specification/common-types/resource-management/v3/types.json#L300